frontend/src/pages/dashboard/jios/forms/JiosCreateEditForm.tsx

```diff
@@ -113,15 +125,40 @@ export default function JiosCreateEditForm({
   return (
     <Box
       sx={{
-        pt: 5,
         pb: 25,
-        px: '15px',
         maxWidth: 600,
         margin: '0 auto',
       }}
     >
+      <AppBar
+        sx={{
+          width: '100vw',
+          maxWidth: 600,
+          mx: 'auto',
+          left: 0,
+          right: 0,
+          background: 'white',
+          color: 'text.primary',
+        }}
+        position="absolute"
+      >
+        <Toolbar sx={{ pl: 1 }}>
+          <IconButton
+            sx={{ mr: 1 }}
+            color="primary"
+            onClick={() => {
+              navigate(-1);
```

Comment:
Perhaps you could extract this into a component

Reply:
fixed